src/System.Data.SqlClient/ref/System.Data.SqlClient.cs

```diff
@@ -478,6 +478,9 @@ public sealed partial class SqlConnection : System.Data.Common.DbConnection, Sys
         public override void Close() { }
         public new System.Data.SqlClient.SqlCommand CreateCommand() { throw null; }
         protected override System.Data.Common.DbCommand CreateDbCommand() { throw null; }
+        public override DataTable GetSchema() { throw null; }
```

Comment:
As a convention fully use the fully qualified name of the DataType. 
`DataTable` should be `System.Data.DataTable`